tests/kola/podman/dns/test.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+ok() {
+    echo "ok" "$@"
+}
+
+fatal() {
+    echo "$@" >&2
+    exit 1
+}
+
+sudo su - tester
+podman network create testnetwork
+
+OUTPUT=$(podman run --rm -t --network=testnetwork registry.fedoraproject.org/fedora:34 getent hosts google.com)
+
+if [[ ${OUTPUT} != "2607:f8b0:4005:807::200e google.com"* ]]; then
+    fatal "DNS in rootless podman testnetwork failed. Test Fails"
+else
+    ok "DNS in rootless podman testnetwork Suceeded. Test Passes"
+fi
```

Comment:
I think checking the output of the command is not ideal here (because the IP address can change). It would be better to just check to see if the command was successful or not.

```suggestion
if !  podman run --rm -t --network=testnetwork registry.fedoraproject.org/fedora:34 getent hosts google.com; then
    fatal "DNS in rootless podman testnetwork failed. Test Fails"
else
    ok "DNS in rootless podman testnetwork Suceeded. Test Passes"
fi
```


Let's also delete the `testnetwork` after the test is done.